src/components/UserPlan.tsx

```diff
@@ -1,42 +1,207 @@
+          openModalHandler={openModalHandler}
+        >
+          <FormattedMessage id="header.send_request" />
+          <div className="tooltiptext">
+            <FormattedMessage id="header.tooltip_last_plan" />
+          </div>
+        </OpenModalButton>
+        <span className="ms-icon icon-info-icon"></span>
+      </Tooltip>
+    </div>
+  );
+
+  const renderPlanLink = () => {
+    if (buttonUrl && !pendingFreeUpgrade)
+      return (
+        <a className="user-plan" href={buttonUrl}>
+          {buttonText}
+        </a>
+      );
+  };
+
   return (
     <div className="user-plan--container">
-      <div className="user-plan--type">
-        <p className="user-plan--monthly-text">
-          <strong>{planName}</strong> ({maxSubscribers} {itemDescription})
-        </p>
-        {buttonUrl && !pendingFreeUpgrade && (
-          <a className="user-plan" href={buttonUrl}>
-            {buttonText}
-          </a>
-        )}
-        <p className="user-plan--monthly-text" />
-      </div>
-
-      <div className="user-plan--type">
-        <div className="user-plan--buyContainer">
-          <p>
-            <strong>{remainingCredits}</strong> {description}
-          </p>
-        </div>
-      </div>
+      {!hasClientManager && (
+        <>
+          <UserPlanType>
+            {isSubscribers || isMonthlyByEmail ? (
+              <>
+                <MonthlyPlan>
+                  <strong>{planName}</strong> ({maxSubscribers}{" "}
+                  {itemDescription})
+                </MonthlyPlan>
+                {renderPlanLink()}
+              </>
+            ) : (
+              <>
+                <MonthlyPlan>
+                  <FormattedMessage id="header.plan_prepaid" />
+                </MonthlyPlan>
+                {renderPlanLink()}
+              </>
+            )}
+            {!buttonUrl || pendingFreeUpgrade
+              ? !isLastPlanRequested
+                ? renderModalButton()
+                : renderModalButtonWithTooltip()
+              : ""}
+          </UserPlanType>
+          <UserPlanType>
+            {isPlanTypeMonthlyDeliveries || isPlanTypeSuscribers ? (
+              <BuyContainer>
+                <p>
+                  {Number(maxSubscribers) - Number(remainingCredits)}{" "}
+                  {isPlanTypeSuscribers ? (
+                    <FormattedMessage id="header.plan_suscribers" />
+                  ) : (
+                    <FormattedMessage id="header.plan_emails" />
+                  )}{" "}
+                  (<strong>{remainingCredits}</strong>{" "}
+                  <FormattedMessage id="header.availables" />)
+                </p>
+              </BuyContainer>
+            ) : (
+              <BuyContainer>
+                <p>
+                  <strong>{remainingCredits}</strong> {description}
+                </p>
+              </BuyContainer>
+            )}
+            {!!Object.keys(sms).length && smsEnabled && (
```

Comment:
En general tratamos de usar el ternario para evitar confusiones. Más detalles: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
No tengo opinión fuerte, gracias por la explicación.